R/calculateUnifrac.R

```diff
@@ -162,22 +166,27 @@ setMethod("calculateUnifrac",
             links <- colLinks(x)
         }
         # Remove those links (make them NA) that are not included in this tree
-        links[ links$whichTree != tree_name, ] <- NA
+        links[ links$whichTree != tree.name, ] <- NA
         # Take only nodeLabs
         links <- links[ , "nodeLab" ]
-        res <- calculateUnifrac(mat, tree = tree, nodeLab = links, ...)
-        return(res)
+        calculateUnifrac(mat, tree = tree, node.lab = links, ...)
```

Comment:
It seems that your PR has removed all the changes in devel branch. This line has been changed

Reply:
I think you can manually revert your changes regarding changes that removed modification from devel

Reply:
there are not too many